test/test.js

```diff
@@ -15,6 +15,21 @@ describe('background-process', function() {
   beforeEach(mkdirp.bind(mkdirp, actual()));
   afterEach(del.bind(del, actual()));
 
+  it('should return the child process id', function() {
+    this.timeout(10000);
+    var stdout = fs.openSync(actual('stdout.txt'), 'a');
+    var stderr = fs.openSync(actual('stderr.txt'), 'a');
+
+    // start the worker and pass options
+    const childPid = background.start(fixtures('worker.js'), {
+        stdio: [stdout, stderr]
+    });
+
+    if (typeof(childPid) !== Number || !childPid) {
```

Comment:
Since we're using `mocha` to run the tests and `assert` for validating values, you can change this `if` statement into the following 2 lines:

```js
assert(childPid, 'expected to return the child process ID');
assert.equal(typeof childPid, 'number');
```